### PR TITLE
Makefile: Stop pinning calicoctl node run

### PR DESCRIPTION
**Pull request by tomdee**

### CODE REVIEW

File: Makefile.calicoctl [hunk 1]
```diff
@@ -30,7 +29,6 @@ LOCAL_USER_ID?=$(shell id -u $$USER)
 PACKAGE_NAME?=github.com/projectcalico/calicoctl
 
 LDFLAGS=-ldflags "-X $(PACKAGE_NAME)/calicoctl/commands.VERSION=$(CALICOCONTAINERS_VERSION) \
-	-X $(PACKAGE_NAME)/calicoctl/commands/node.VERSION=$(CALICOCTL_NODE_VERSION) \
```
Comment (robbrockbank) [inline Makefile.calicoctl]: You'll need to update the `run.go` to have a hard coded version of latest.

**Comment by robbrockbank** [comment]
Looks fine, but needs a corresponding change to run.go to hard code the node image name.

File: Makefile.calicoctl (hunk 1 above)
Comment (tomdee) [inline Makefile.calicoctl]: Ooops, I forgot to add that file. Done now.

**Comment by robbrockbank** [comment]
LGTM

**Pull request merged**